browser/src/main.js

```diff
@@ -78,7 +78,6 @@ window.addEventListener('beforeunload', function () {
 	}
 });
 
-window.docPermission = permission;
```

Comment:
so we will ignore completly `&permission=xxxxx` from the URL and use only WOPI prop?
what about existing integrators? They can use that still

Reply:
thanks for the catch, I forgot about URL param